Adjust path to accerciser's modules to look in vendor-packages instead
of in site-packages, to match where we install on Solaris.

Not suitable for upstream as-is, would need modification to follow
$(pkgpythondir) from configure instead.

--- accerciser-3.42.0/src/accerciser.in
+++ accerciser-3.42.0/src/accerciser.in
@@ -40,7 +40,7 @@ # We can't rely on prefix if we're insta
 # use __file__ and for now hope that lib is relative to bin.
 sys.prefix = '@prefix@'
 libs = os.path.join(sys.prefix, 'lib',
-                    'python@PYTHON_VERSION@', 'site-packages')
+                    'python@PYTHON_VERSION@', 'vendor-packages')
 # point to the proper site-packages path
 sys.path.insert(1, libs)
 
